stripe/src/main/java/com/stripe/android/view/PaymentAuthWebViewActivity.java

```diff
@@ -49,4 +72,23 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
         return super.onOptionsItemSelected(item);
     }
+
+    void customizeToolbar(@NonNull Toolbar toolbar) {
+        if (mToolbarCustomization != null) {
+            if (!StripeTextUtils.isBlank(mToolbarCustomization.getHeaderText())) {
+                toolbar.setTitle(CustomizeUtils.buildStyledText(this,
+                        mToolbarCustomization.getHeaderText(), mToolbarCustomization));
+            }
+
+            if (mToolbarCustomization.getBackgroundColor() != null) {
+                @ColorInt final int backgroundColor =
+                        Color.parseColor(mToolbarCustomization.getBackgroundColor());
+                toolbar.setBackgroundColor(backgroundColor);
+
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+                    getWindow().setStatusBarColor(CustomizeUtils.darken(backgroundColor));
```

Comment:
optional: maybe a better API for `CustomizeUtils.darken()` would be to pass in the activity as well and do the API version check inside the `darken` method?

Reply:
Will update this in a future PR!